src/rpc/blockchain.cpp

```diff
@@ -18,6 +18,8 @@
 #include <hash.h>
 #include <index/blockfilterindex.h>
 #include <index/coinstatsindex.h>
+#include <net.h> // For NodeId
```

Comment:
nit: not sure if there is need for a `// For NodeId` comment here. I think comments like these tend to be overlooked when adding something else that needs `net.h` and thus out-dating the comment.

Reply:
Yea, please don't add comments like these.